cookbooks/aws-parallelcluster-scheduler-plugin/recipes/config_head_node.rb

```diff
@@ -27,3 +27,28 @@
 execute_event_handler 'HeadConfigure' do
   event_command(lazy { node['cluster']['config'].dig(:Scheduling, :SchedulerSettings, :SchedulerDefinition, :Events, :HeadConfigure, :ExecuteCommand, :Command) })
 end
+
+unless virtualized?
```

Comment:
this 'initialize compute fleet status in DynamoDB' is already present in https://github.com/aws/aws-parallelcluster-cookbook/pull/1305/files